### ISSUE TITLE
Issue about Page admin/administration-panel/Studio.adoc

### ISSUE THREAD

**Issue by tabularjon** [salesagility/SuiteDocs#315]
#### Page with Issue
https://docs.suitecrm.com/admin/administration-panel/studio/

#### What is wrong, less clear, or missing
Adding A Dropdown Field

Unhelpful list of drop-downs. It should include a blank item and any other special cases. For example a common form of list start with  Select >> linking to a blank item. This ensures that if user does nothing we do not get a false value in the database.

**Comment by pgorod** [comment]
Hi. I'm not sure how SuiteCRM handles those blank values and other dropdown validations, but just to clarify: your objection is about the way the documentation explains behavior, or about the actual behavior of the app?

**Comment by tabularjon** [comment]
Hi Pgorod,

Thanks for getting back so quickly.

I thought if the documentation contained information on the common requirement of adding a first/default blank entry to a dropdown list, it would be very helpful. As it is, I couldn’t find this information anywhere and so have to guess a bit. Not great really. So instead of just a useless list of colours, it could include a more real-world example with a default blank entry and how to create a blank entry (which is actually a lot simpler than it seems).

For example I tried entering [-blank] and [Select] but it didn’t work. So you actually have to enter [] (nothing) and [Select] for it to appear as [-blank-] and [Select]. I believe there is also a way of denoting that an item is the default entry, again this would be really useful to show. 

I am fine with having a dumbed down version in the Documentation if there is an easy link on it to click for advanced options, but I could not find this. The documentation should be complete and show all options, not just the easy ones, in my opinion.
Kind regards,

Jon 



Jonathan Bearman
Head of IT
Maths — No Problem!

Office: 01892 537706  Direct: 01892 577101  Mobile: 07958 663493
www.mathsnoproblem.com <http://mathsnoproblem.com/>
 <https://twitter.com/MathsNoProblem>  <https://www.instagram.com/mathsnoproblem/>  <https://www.facebook.com/MathsNoProblem/>
Proud winners of:
The IPG Independent Publishing Awards, 2017–2018
The British Book awards, 2018

**Comment by pgorod** [comment]
I can add an entry with blank name and blank text, but I can't add an entry with blank name and `[Select]` as text:

![image](https://user-images.githubusercontent.com/15945027/62119574-cd061d00-b2b7-11e9-9eef-e5e4ec27e852.png)

By the way, you can reorder the list and the first one will be (I think) the default. We can add this to the docs to make it clearer.

**Comment by tabularjon** [comment]
Thanks Pgorod,

You are right, although I entered Select with -blank-, this did not save as such — so it doesn’t appear possible to do this, which is a shame. The documentation would make this clear, ideally.

I found the default can be changed but not on the dropdown editor page, it has to be done on the field editor page.

Kind regards,
Jon 



Jonathan Bearman
Head of IT
Maths — No Problem!

Office: 01892 537706  Direct: 01892 577101  Mobile: 07958 663493
www.mathsnoproblem.com <http://mathsnoproblem.com/>
 <https://twitter.com/MathsNoProblem>  <https://www.instagram.com/mathsnoproblem/>  <https://www.facebook.com/MathsNoProblem/>
Proud winners of:
The IPG Independent Publishing Awards, 2017–2018
The British Book awards, 2018
